assets/stylesheets/_components.scss

```diff
@@ -177,6 +172,11 @@
 @import 'my-sites/draft/style';
 @import 'my-sites/drafts/style';
 @import 'my-sites/exporter/style';
+@import 'my-sites/invites/invite-form-header/style';
+@import 'my-sites/invites/invite-header/style';
+@import 'my-sites/invites/invite-accept/logged-in/style';
```

Comment:
A subcomponent is `invite-accept/logged-in.jsx`, the folder `logged-in` is a full component.